lib/WP_Auth0_Nonce_Handler.php

```diff
@@ -12,7 +12,7 @@
   /**
    *
    */
-  const COOKIE_EXPIRES = MINUTE_IN_SECONDS;
+  const COOKIE_EXPIRES = HOUR_IN_SECONDS;
```

Comment:
Longer state cookie, was failing if you left the login page open for more than a minute.